lib/Table.js

```diff
@@ -162,8 +162,9 @@ Table.prototype.init = function (next) {
 
         } else {
           if (indexes.delete.length > 0 || indexes.create.length > 0) {
-            debug('indexes are not synchronized and update flag is set to false');
-            deferred.reject(new Error('indexes are not synchronized and update flag is set to false'));
+            var mess = table.name + 'indexes are not synchronized and update flag is set to false';
```

Comment:
Can we add a space before the `indexes` word so there is a space between the table name and indexes?